ale_linters/javascript/eslint.vim

```diff
@@ -25,14 +25,10 @@ function! ale_linters#javascript#eslint#Handle(buffer, lines)
             continue
         endif
 
-        let text = l:match[3]
         let marker = l:match[4]
         let marker_parts = split(marker, '/')
         let type = marker_parts[0]
-
-        if len(marker_parts) == 2
-            let text = text . ' (' . marker_parts[1] . ')'
-        endif
+        let text = l:match[3] . ' (' . marker_parts[1] . ')'
```

Comment:
I think the length here is sometimes `2`, and sometimes `1`, because you sometimes get something like `Warning/comma-dangle` and at other times just `Error`. I'd probably leave this part as it is.